src/components/PlioQuestion.vue

```diff
@@ -218,13 +227,88 @@ export default {
           this.hideSkipButton();
           this.isAnswerSubmitted = true;
           this.disableRadioButtons();
-        }, 200);
+        });
       }
 
       this.text = "";
       this.show = true;
       document.querySelector("body").classList.add("overflow-hidden");
       this.newProgressBarInfo['progressPercent'] = this.progressBarInfo['progressPercent']
+      this.handleImage()
+      this.styleQuestion()
+      this.styleOptions()
+      this.renderContent()
+    },
+
+    handleImage(){
+      this.$nextTick(() => {
+        // get question element
+        var question = document.getElementById('question')
+        if (question == null) return
+        // extract all <p> tags from the question element
+        var p_tags = question.querySelectorAll('p')
+        if (p_tags == null || p_tags.length == 0) return
+        // find which <p> tag has an <img> tag, cut it and
+        // paste it right outside, just below it.
+        p_tags.forEach(p_tag => {
+          var img_tag = p_tag.querySelector('img')
+          if (img_tag != null) {
+            p_tag.removeChild(img_tag)
+            p_tag.parentNode.insertBefore(img_tag, p_tag.nextSibling)
+            img_tag.setAttribute("id", "question_image")
```

Comment:
Removes the `img` from the `p` tag and places it just outside, below the same `p` tag.
Note: in JS, there's no `insertAfter` method. Hence there is a workaround using the `insertBefore` method. More details [here](https://www.javascripttutorial.net/javascript-dom/javascript-insertafter/)

Reply:
Why is this needed?

Reply:
The `<img>` tag is a child of the `<p>` tag. I took it out and placed it next to the `<p>` tag so I'm able to manipulate it easily.
The parent of `<p>` tag has `flex` display property. To align the image in the center, it needs to be taken out. 
```
flex-direction: column;
align-items: center;
```
The above piece of code helped center the image independent of the `<p>` tag

Reply:
Hmm... It wasn't possible to insert after the previous `p` tag? 
Anyway, this not a big deal. Can you add a comment in the code?